### PR TITLE
Add archives section to scholarX 2020 page

### LINKED ISSUE #539: Add a new section on the 2020 scholarX page to display past scholarX programs
**Is your feature request related to a problem? Please describe.**
- Currently you can't check past year scholarx pages.

**Describe the solution you'd like**
- Add a new section on the 2020 scholarX page.
Screenshot from 2020-02-27 20-01-40
- use `../past-scholarx.html#past-scholarx` for archive link

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #539 

## Goals

- Add a new section on the 2020 scholarX page.

## Approach

- Add a new section on the 2020 scholarX page.
- Use ../past-scholarx.html#past-scholarx for archive link

### Screenshots

Screenshot from 2020-02-28 13-11-36

  
### Preview Link

https://pr-548-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/scholarx/2020/index.html [hunk 1]
```diff
@@ -482,6 +482,25 @@ <h5 class="mb-0">
...
+    <div id="past-onelives" class="container">
+        <div class="row justify-content-center">
+            <div class="col-9 text-center">
+                <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+                <p class="display-3 text-onelive font-weight-bolder">ScholarX</p>
+            </div>
+        </div>
+        <div class="row">
+        </div>
+        <div class="row justify-content-center mb-md-5">
```
Comment (Gravewalker666) [inline src/main/webapp/scholarx/2020/index.html]: What is this extra row?

**Review by Gravewalker666 — DISMISSED** [review]

File: src/main/webapp/scholarx/2020/index.html (hunk 1 above)
Comment (anjula-sack) [reply src/main/webapp/scholarx/2020/index.html]: I fixed this.

File: src/main/webapp/scholarx/2020/index.html [hunk 2]
```diff
@@ -482,6 +482,24 @@ <h5 class="mb-0">
...
+        <div class="row justify-content-center">
+            <div class="col-9 text-center">
+                <h1 class="mb-0 text-onelive">Take a look at our past</h1>
+                <p class="display-3 text-onelive font-weight-bolder">ScholarX</p>
+            </div>
+        </div>
+        </div>
+        <div class="row justify-content-center mb-md-5">
+            <div class="col-9 text-center">
+                <a href="../past-scholarx.html#past-scholarx"
```
Comment (jayasanka-sack) [inline src/main/webapp/scholarx/2020/index.html]: The correct link should be  `<a href="../archives`

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Change the commit message and the PR title it should be something like, "Add archives section to scholarX 2020 page"

File: src/main/webapp/scholarx/2020/index.html (hunk 2 above)
Comment (anjula-sack) [reply src/main/webapp/scholarx/2020/index.html]: I resolved it.

**Pull request closed without merging**